index.bs

```diff
@@ -1767,6 +1771,9 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 1. If the length of <code>|pkOptions|.{{PublicKeyCredentialCreationOptions/user}}.{{PublicKeyCredentialUserEntity/id}}</code> is not between 1 and 64 bytes (inclusive) then throw a {{TypeError}}.
 
+1. If <code>|pkOptions|.{{PublicKeyCredentialCreationOptions/user}}.{{PublicKeyCredentialUserEntity/language}}</code> is present
+    and is not a [=well-formed language tag=], throw a "{{SyntaxError}}" {{DOMException}}.
```

Comment:
WebAuthn typically raises `TypeError`'s when there's an issue with a malformed value, as documented in L3:

- https://w3c.github.io/webauthn/#sctn-create-request-exceptions
- https://w3c.github.io/webauthn/#sctn-get-request-exceptions

We should probably maintain this behavior for this exception too.

Reply:
Hm. I see what you mean, but `TypeError` is not semantically correct for this error case. The `user.id` error ideally should have been a `SyntaxError` too, but that's probably too late to change now. [`SyntaxError` is also used by the PRF extension when base64 decoding fails](https://www.w3.org/TR/2025/WD-webauthn-3-20250127/#ref-for-dom-authenticationextensionsprfinputs-evalbycredential%E2%91%A2). So I'd prefer to keep this as `SyntaxError` rather than `TypeError`.